docs/data/remote-examples/language-examples/examples.yaml

```diff
@@ -20,9 +20,11 @@ paths:
         - solr
     nodejs:
         - elasticsearch
+        - influxdb
         - memcached
         - mongodb
         - mysql
+        - opensearch
```

Comment:
:question: What's the benefit of adding the examples to the site if they're not used?